waiter/integration/waiter/basic_test.clj

```diff
@@ -820,7 +820,8 @@
 
 (deftest ^:parallel ^:integration-fast test-basic-priority-support
   (testing-using-waiter-url
-    (let [headers {:x-waiter-name (rand-name)
+    (let [headers {:x-waiter-concurrency-level 1
```

Comment:
Avoids any issues due to concurrent processing of requests if we ever change the defaults.